src/lib/structures/Precondition.ts

```diff
@@ -25,5 +25,5 @@ export abstract class Precondition extends Piece {
 }
 
 export interface PreconditionContext extends Record<PropertyKey, unknown> {
-	command: Command;
+	external?: boolean;
```

Comment:
huh? That's a completely different name and type

Reply:
oh fuck this wasn't meant to be a requested change but just a comment

Reply:
![image](https://user-images.githubusercontent.com/24852502/107149637-f34f5300-6959-11eb-8384-e558d664b6e0.png)